stable/aad-pod-identity/README.md

```diff
@@ -0,0 +1,139 @@
+> __NOTE:__ It's simpler to use the same resource group as your Kubernetes nodes are deployed in. For AKS this is the MC_* resource group. If you can't use the same resource group, you'll need to grant the Kubernetes cluster's service principal the "Managed Identity Operator" role.
+```shell
+az identity create -g <resource-group> -n <id-name>
+```
+
+2. Assign your newly created identity the appropriate role to the resource you want to access.
+</details>
+
+
+#### Installing charts
+
+* If you need the azure identity and azure identity binding resources also to be created as part of the chart installation, update the values.yml to enable the azureIdentity and replace the resourceID, clientID using the values for the user identity.
+* If you need the aad-pod-identity deployment to use it's own service principal credentials instead of the cluster service prinicipal '/etc/kubernetes/azure.json`, then uncomment this section and add the appropriate values for each required field.
+
+```
+adminsecret:
+  cloud: <cloud environment name>
+  subscriptionID: <subscription id>
+  resourceGroup: <cluster resource group>
+  vmType: <`standard` for normal virtual machine nodes, and `vmss` for cluster deployed with a virtual machine scale set>
+  tenantID: <service principal tenant id>
+  clientID: <service principal client id>
+  clientSecret: <service principal client secret>
+```
+
+To install the chart with the release name `my-release`:
+
+```console
+$ helm install --name my-release stable/aad-pod-identity
+```
+
+Deploy your application to Kubernetes. The application can use [ADAL](https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-authentication-libraries) to request a token from the MSI endpoint as usual. If you do not currently have such an application, a demo application is available [here](https://github.com/Azure/aad-pod-identity#demo-app). If you do use the demo application, please update the `deployment.yaml` with the appropriate subscription ID, client ID and resource group name. Also make sure the selector you defined in your `AzureIdentityBinding` matches the `aadpodidbinding` label on the deployment.
+
+## Uninstalling the Chart
+
+To uninstall/delete the last deployment:
+
+```console
+$ helm ls
+$ helm delete [last deployment]
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+CRDs created by this chart are not removed by default and should be manually cleaned up:
+```bash
+kubectl delete crd azureassignedidentities.aadpodidentity.k8s.io
+kubectl delete crd azureidentities.aadpodidentity.k8s.io
+kubectl delete crd azureidentitybindings.aadpodidentity.k8s.io
+kubectl delete crd azurepodidentityexceptions.aadpodidentity.k8s.io
+```
+
+## Known Issues
+
+__Error Redeploying Chart__
+
+If you have previously installed the helm chart, you may come across the following error message:
```

Comment:
Can you try using helm version `2.14.2`? This version addresses this particular issue.